Packs/FortiSIEM/ReleaseNotes/2_0_37.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### FortiSIEM v2
+
+- The **Fetch with Events**§ fetch mode will now work only for FortiSiem version 6.6 and earlier.
+- The ***fortisiem-event-search*** command will now work only for FortiSiem version 6.6 and earlier.
```

Comment:
```suggestion
- Updated the integration documentation to clarify that the **Fetch with Events** fetch mode, as well as the ***fortisiem-event-search*** command, are supported only for FortiSiem version 6.6 and earlier due to backward compatibility changes in subsequent versions of FortiSiem.
```
it's not "will now only work", since we didn't change anything